src/main.rs

```diff
@@ -54,8 +55,9 @@ fn main() {
         ("serve", Some(matches)) => {
             let interface = matches.value_of("interface").unwrap_or("127.0.0.1");
             let port = matches.value_of("port").unwrap_or("1111");
+            let public = matches.value_of("output_dir").unwrap_or("public");
```

Comment:
same as above

Reply:
I followed the same structure as :

### interface
https://github.com/Keats/gutenberg/blob/master/src/cli.rs#L38
https://github.com/Keats/gutenberg/blob/master/src/main.rs#L55

### port

https://github.com/Keats/gutenberg/blob/master/src/cli.rs#L43
https://github.com/Keats/gutenberg/blob/master/src/main.rs#L56

Please advice

Reply:
I'm just not sure how the default works, I guess you can try removing the `unwrap_or` and see if it still works?

Reply:
Without the `unwrap_or`, the type goes from `&str` to `Option<&str>`.
It could be made working with `if let Some(x) = ..{}`, but IMHO that would be less clearer then the `unwrap_or`.

What might be interesting, to make the `main.rs` more uniform, is make the same `unwrap_or` for `base_url` change:
* https://github.com/Keats/gutenberg/blob/master/src/main.rs#L46
* https://github.com/Keats/gutenberg/blob/master/src/cmd/build.rs#L10-L12

Please advice.

Reply:
>  the type goes from &str to Option<&str>

I guess with a default, it can never be `None` right?